apps/src/lib/types/mod.rs

```diff
@@ -1,10 +1,8 @@
 use std::collections::HashSet;
 
-use crate::proto::types;
-
 #[derive(Debug)]
 pub enum MatchmakerMessage {
-    InjectTx(types::Tx),
+    InjectTx(Vec<u8>),
```

Comment:
Maybe this message could be rename to `CraftAndInjectTx`, but it might be a bit too long ?

Reply:
Yeah, I think that's a bit wordy and I'm not sure if it actually makes it clearer. Let's give a more detailed description in docstrings (#258).